README.md

```diff
@@ -12,6 +12,7 @@ Lifecycle Configurations provide a mechanism to customize Notebook Instances via
 * [connect-emr-cluster](scripts/connect-emr-cluster) - This script connects an EMR cluster to the Notebook Instance using SparkMagic.
 * [disable-uninstall-ssm-agent](scripts/disable-uninstall-ssm-agent) - This script disables and uninstalls the SSM Agent at startup.
 * [execute-notebook-on-startup](scripts/execute-notebook-on-startup) - This script executes a Notebook file on the instance during startup.
+* [export-to-pdf-fix](scripts/export-to-pdf-fix) - This script fixes a problem in SageMaker Notebook instances where Jupyter fails to export a notebook directly to PDF.
```

Comment:
Do you think `export-to-pdx-enable` is a better name? :-)

Reply:
Good idea. I've resolved it.